product_docs/docs/pgd/5/ddl.mdx

```diff
@@ -177,6 +186,10 @@ can be reached and are keeping up reasonably well with the current write rate.
 If such DDL commands must be performed while a node is down, first remove the
 down node from the configuration.
 
+A global DML lock is granted when *ALL* the eligible nodes, data and witness, agree to grant it.
+Subscriber-only nodes are not eligible to participate.
+
```

Comment:
I looked at the code for this - and also the ticket mentions it - witness nodes do not contribute to DML locks. They participate in DDL and advisory locks but not DML. Therefore this should be:
A global DML lock is granted when *ALL* the eligible data nodes agree to grant it.
Subscriber-only nodes and witness nodes are not eligible to participate.

Reply:
bdr_gl_node_participates_in_locking